src/shared/stores/__tests__/shareModal.tests.js

```diff
@@ -0,0 +1,137 @@
+import base from '../index';
+import shareModal from '../shareModal';
+
+describe('Theme › Shared › Stores › ShareModal', () => {
+  test('Has `isOpen`, `isLinkCopied` and `item` props', () => {
+    const self = shareModal.create();
+
+    expect(self.isOpen).not.toBe(undefined);
```

Comment:
Maybe, in general, we can skip these "notToBeUndefineds" because they are covered by the snapshot. What do you think?

Reply:
I mean if they disappear (which they shouldn't) the tests will fail anyway because the snapshot did change.

Reply:
I thought about that, and I think yes, we can remove them. I just left them because I don't trust snapshots 😆

Reply:
Maybe with time... 😆